organizations.go

```diff
@@ -0,0 +1,11 @@
+package datadog
+
+type OrganizationResponse struct {
+	Data *Organization `json:"data,omitempty"`
+}
+
+// TODO: Fill this out with everything you can get back when getting an organization
```

Comment:
❓

Reply:
This was just a comment that there's much more to Organizations than an ID, but that's all I needed for this implementation. I can easily delete the comment or create an issue for adding Organizations support.